qubits/example_qubit.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jul 26 18:18:15 2022
+
+@author: Theo
+"""
+
+import numpy as np
+from component import Component
+import sdxf
+from pt_operations import rotate_pt, rotate_pts, translate_pts, arc_pts, orient_pts, mirror_pts, translate_pt, orient_pt
+from junction import junction
+from component import Component
+
+from qubits.notch import QubitNotchFromJunc
+
+class ExampleQubit(Component):
+    """
+    
+    """
+    _defaults = {}
+    _defaults['pinw'] = 20
+    _defaults['gapw'] = 8.372
+    _defaults['refjunc'] = None
+    _defaults['paddle_width'] = 20
+    _defaults['paddle_length'] = 100
+    _defaults['paddle_gap'] = 5
+    _defaults['taper_angle'] = 60
+    _defaults['h_padding'] = 10
+    _defaults['v_padding'] = 20
+    _defaults['offset'] = 0
+    _defaults['pin_gap'] = 8.372
+    _defaults['leftright'] = 'right'
+    
+    def __init__(self,structure,settings):
+        
+        s = structure
+        
+        comp_key = 'ExampleQubit'
+        global_keys = ['pinw','gapw','pin_gap']
+        object_keys = ['pinw','gapw','gapw'], # which correspond to the extract global_keys
+        Component.__init__(self,structure,comp_key,global_keys,object_keys,settings)
+             
+        coords = self.refjunc.coords
+        direction = self.refjunc.direction + 180
+        
+        radius = self.paddle_width/2
+        
```

Comment:
try to add structure comments, i.e. "drawing the qubit paddles", "drawing the notch" etc.